### PR TITLE
Rest api tests

### PR DESCRIPTION
**Pull request by GiorgioJovanni**

### CODE REVIEW

File: Tests/Tests_REST_API/TestData.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    customerId = 12656
+    amount = 5000
+    new_account_id = randint(10000, 99999)
+    newAccountType = 0
+    username = 'ivan'
+    password = 'ivan123'
+
+
+class DataEndPoints:
+    base_uri = 'https://parabank.parasoft.com/parabank/services/bank'
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/TestData.py:14]: Don't put data like DataTest.customerId inside endpoint, add this data when you create test

File: Tests/Tests_REST_API/test_base_page.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+from Tests.Tests_REST_API.BaseClass import ParaBankHomePage
+from Tests.Tests_REST_API.TestData import DataEndPoints
+
+
+class TestParaBank(ParaBankHomePage):
+
+    def test_get_account_by_id(self):
+        response = self.get(f"{DataEndPoints.base_uri}{DataEndPoints.getAccountId}")
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py]: please write this like DataEndPoints.base_uri + DataEndPoints.getAccountId
it is more readable

File: Tests/Tests_REST_API/test_base_page.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        assert response.status_code == 200, "           Can't get response 200 for account by id"
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py:9]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/test_base_page.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+    def test_get_list_transaction_for_account(self):
+        response = self.get(f"{DataEndPoints.base_uri}{DataEndPoints.getListTransactionForAccount}")
+        assert response.status_code == 200, "           Can't get response 200 for the list transaction for the account"
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py:15]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/test_base_page.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    def test_get_create_transactions_by_amount_for_account(self):
+        response = self.get(f"{DataEndPoints.base_uri}{DataEndPoints.createTransactionsByAmountForAccount}")
+        assert response.status_code == 200, "           Can't get response 200 for create " \
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/test_base_page.py [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+                                            "transactions with amount for account"
+
+    def test_post_new_deposit(self):
+        response = self.post(f"{DataEndPoints.base_uri}{DataEndPoints.newDeposit}")
+        assert response.status_code == 200, "           Can't get response 200 for new deposit"
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py:21]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/test_base_page.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+    def test_post_new_account(self):
+        response = self.post(f"{DataEndPoints.base_uri}{DataEndPoints.newAccount}")
+        # print(response.text)
+        assert response.status_code == 200, "           Can't get response 200 for new account"
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/test_base_page.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+    def test_get_login(self):
+        response = self.get(f"{DataEndPoints.base_uri}{DataEndPoints.login}")
+        assert response.status_code == 200, "           Can't get response 200 for login"
```
Comment (AramaisTyshchenko) [inline Tests/Tests_REST_API/test_base_page.py]: you need to verify incoming data response.json()

File: Tests/Tests_REST_API/TestData.py (hunk 1 above)
Comment (GiorgioJovanni) [reply Tests/Tests_REST_API/TestData.py:14]: Done

File: Tests/Tests_REST_API/test_base_page.py (hunk 3 above)
Comment (GiorgioJovanni) [reply Tests/Tests_REST_API/test_base_page.py:9]: Done

**Pull request closed without merging**